hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -179,10 +179,14 @@ ContainerDataProto readContainer(long containerID)
    * @return ContainerInfo
    * @throws IOException - in case of error.
    */
+  @Deprecated
   ContainerWithPipeline createContainer(HddsProtos.ReplicationType type,
       HddsProtos.ReplicationFactor replicationFactor,
       String owner) throws IOException;
 
+  ContainerWithPipeline createContainer(ReplicationConfig replicationConfig,
+                                        String owner) throws IOException;
```

Comment:
nit: Please do not format method signature like this. Whenever visibility / return type / method name / other modifiers are changed, we would have to reindent all parameters.